nibabel/nicom/dicomwrappers.py

```diff
@@ -461,10 +461,23 @@ def __init__(self, dcm_data):
         Wrapper.__init__(self, dcm_data)
         self.dcm_data = dcm_data
         self.frames = dcm_data.get('PerFrameFunctionalGroupsSequence')
+        self._nframes = self.get('NumberOfFrames')
         try:
             self.frames[0]
         except TypeError:
             raise WrapperError("PerFrameFunctionalGroupsSequence is empty.")
+        try:
+            # DWI image where derived isotropic, ADC or trace volume
+            # was appended to the series
+            if self.frames[0].get([0x18, 0x9117], None):
```

Comment:
Looks like `None` is the default anyway.

```suggestion
            if self.frames[0].get([0x18, 0x9117]):
```